core/src/main/java/org/eclipse/hono/util/TenantObject.java

```diff
@@ -117,112 +131,90 @@ public TenantObject setEnabled(final boolean flag) {
     }
 
     /**
-     * Gets the subject DN of this tenant's configured trusted
-     * certificate authority.
+     * Get the subject DN of the configured trusted CA.
      * 
      * @return The DN or {@code null} if no CA has been set.
      */
     @JsonIgnore
+    @Deprecated
```

Comment:
FMPOV we should simply remove this method

Reply:
sure, its also not used anyway